modules/SDL.lua

```diff
@@ -573,6 +573,35 @@ function SDL.AppInfo.clean()
   deleteFile(SDL.AppInfo.file())
 end
 
+SDL.HMICapCache = {}
+
+function SDL.HMICapCache.file()
+  return getFilePath(SDL.INI.get("HMICapabilitiesCacheFile"), SDL.INI.get("AppStorageFolder"))
+end
+
+function SDL.HMICapCache.get()
+  local content = getFileContent(SDL.HMICapCache.file())
+  if content ~= nil then return json.decode(content) end
+  return {}
```

Comment:
```suggestion
  return nil
```
I would propose to return nil because the content can be empty

Reply:
@VjKlepikov Done.